manager/src/main/java/com/redhat/service/bridge/manager/connectors/ConnectorsOrchestrateImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.redhat.service.bridge.manager.connectors;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import com.redhat.service.bridge.infra.models.dto.ConnectorStatus;
+import com.redhat.service.bridge.manager.dao.ConnectorsDAO;
+import com.redhat.service.bridge.manager.models.ConnectorEntity;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.eventbus.EventBus;
+import io.vertx.mutiny.core.eventbus.Message;
+
+import static com.redhat.service.bridge.manager.connectors.Events.CONNECTOR_CREATED_EVENT;
+import static com.redhat.service.bridge.manager.connectors.Events.CONNECTOR_DELETED_EVENT;
+import static com.redhat.service.bridge.manager.connectors.Events.KAFKA_TOPIC_CREATED_EVENT;
+import static com.redhat.service.bridge.manager.connectors.Events.KAFKA_TOPIC_DELETED_EVENT;
+
+// Currently disabled (not scheduled).
+// Part of https://issues.redhat.com/browse/MGDOBR-155
+@ApplicationScoped
+public class ConnectorsOrchestrateImpl implements ConnectorsOrchestrate {
+
+    @Inject
+    ConnectorsDAO connectorsDAO;
+
+    @Inject
+    EventBus eventBus;
+
+    @Override
+    @Transactional(Transactional.TxType.NEVER)
+    public Uni<List<Message<ConnectorEntity>>> updatePendingConnectors() {
```

Comment:
I think a pending connector might be also a connector that has the `workerId` set but for any reason that worker did not release the lock (it crashed, it was restarted, any reason). Currently `findUprocessed` is not returning such processors

Reply:
Yes this will be adressed in https://issues.redhat.com/browse/MGDOBR-155